src/QuickMarcEditor/QuickMarcEditorRows/utils.js

```diff
@@ -7,3 +7,7 @@ export const isReadOnly = recordRow => READ_ONLY_ROWS.includes(recordRow.tag);
 const INDICATOR_EXEPTION_ROWS = [LEADER_TAG, '001', '002', '003', '004', '005', '006', '007', '008', '009'];
 
 export const hasIndicatorException = recordRow => INDICATOR_EXEPTION_ROWS.includes(recordRow.tag);
+
+const ADD_EXCEPTION_ROWS = [LEADER_TAG, '001', '005', '008'];
```

Comment:
are you sure 999 should have add button? it can't be edited, removed, but can be duplicated? please clarify with Stef